packaging/package/package.py

```diff
@@ -57,13 +42,30 @@ def main():
         return
 
     if not files_to_package:
-        logger.info("Packaging failed. Check " + args.log_path + " for more information.")
+        logger.info("Packaging failed due to Connector Files not valid. Check " + args.log_path + " for more information.")
```

Comment:
Wording is a bit awkward, I think it is fine the way it was since packaging and signing are two separate steps.

Reply:
done